forms-flow-web/src/components/CustomComponents/settingsModal.js

```diff
@@ -461,7 +461,7 @@ const SettingsModal =
         
         
       </Modal.Body>
-      <Modal.Footer className="d-flex justify-content-start">
+      <Modal.Footer>
```

Comment:
Had checked any breakages by removing this class?

Reply:
Yes. handled in generic modal footer class @Bonymol-aot